### PR TITLE
Move non-bin mod into subdir

### LINKED ISSUE #3141: TiKV needs to reorganize bin directory for auto discovery
Latest Cargo warns: 
``` 
warning: An explicit [[bin]] section is specified in Cargo.toml which currently
disables Cargo from automatically inferring other binary targets.
This inference behavior will change in the Rust 2018 edition and the following
files will be included as a binary target:

* /home/stn/tikv/src/bin/signal_handler.rs
* /home/stn/tikv/src/bin/profiling.rs
* /home/stn/tikv/src/bin/setup.rs

This is likely to break cargo build or cargo test as these files may not be
ready to be compiled as a binary target today. You can future-proof yourself
and disable this warning by adding `autobins = false` to your [package]
section. You may also move the files to a location where Cargo would not
automatically infer them to be a target, such as in subfolders.

For more information on this warning you can consult
https://github.com/rust-lang/cargo/issues/5330
```

### PR DESCRIPTION
**Pull request by csmoe**
Closes #3141

### CODE REVIEW

File: src/bin/tikv-importer.rs [hunk 1]
```diff
@@ -45,11 +45,10 @@ extern crate tikv;
 extern crate toml;
 
 #[cfg(unix)]
-mod profiling;
 #[macro_use]
-mod setup;
-use setup::*;
-mod signal_handler;
+mod utils;
```
Comment (siddontang) [inline src/bin/tikv-importer.rs]: we use `util` mostly.

**Comment by siddontang** [comment]
Rest LGTM

File: src/bin/tikv-importer.rs (hunk 1 above)
Comment (csmoe) [reply src/bin/tikv-importer.rs]: since `tikv::util` is imported here too, how about `bin_util`?

Comment (breeswish) [reply src/bin/tikv-importer.rs]: Can we put this util into a different namespace? or can we use absolute path, like super::util?

**Comment by breeswish** [comment]
LGTM

**Pull request merged**